src/panels/lovelace/cards/hui-media-control-card.ts

```diff
@@ -190,11 +190,15 @@ export class HuiMediaControlCard extends LitElement implements LovelaceCard {
         >
           <div
             class="color-block"
-            style=${styleMap({ "background-color": this._backgroundColor! })}
+            style=${styleMap({
+              "background-color": `${this._backgroundColor}`,
```

Comment:
```suggestion
              "background-color": this._backgroundColor || "",
```

Reply:
Will fix. Wasnt sure if this was better. Less comparison but it leaves the value as undefined which still falls back

Reply:
It has to parse a template every time, and come with the same result, an empty string and it is bad for readability.